components/contribute-cards/EditTierModal.js

```diff
@@ -771,37 +824,35 @@ export function EditTierForm({ tier, collective, onClose }) {
                 )}
               </ModalHeader>
               <ModalBody>
-                <Flex>
+                <ModalSectionContainer>
                   <EditSectionContainer>
                     <FormFields collective={collective} values={values} />
                   </EditSectionContainer>
                   <PreviewSectionContainer>
                     <ContributeCardPreview collective={collective} tier={values} />
                   </PreviewSectionContainer>
-                </Flex>
+                </ModalSectionContainer>
               </ModalBody>
               <ModalFooter isFullWidth dividerMargin="1rem 0">
-                <Flex justifyContent="right" flexWrap="wrap">
+                <EditModalActionsContainer>
                   {isEditing && (
-                    <StyledButton
+                    <DeleteModalButton
                       type="button"
                       data-cy="delete-btn"
                       buttonStyle="dangerSecondary"
-                      mx={2}
                       minWidth={120}
                       onClick={onDeleteTierClick}
                       loading={isDeleting}
                       disabled={isSubmitting || isConfirmingDelete}
                       marginRight="auto"
                     >
                       <FormattedMessage id="actions.delete" defaultMessage="Delete" />
-                    </StyledButton>
+                    </DeleteModalButton>
                   )}
-                  <StyledButton
+                  <ConfirmModalButton
                     type="submit"
                     data-cy="confirm-btn"
                     buttonStyle="primary"
-                    mx={2}
                     minWidth={120}
```

Comment:
Your fix is a valid approach, but just in case you're not aware of it styled-system offers us [responsive props](https://styled-system.com/api#flexbox) based on our [breakpoints](https://github.com/opencollective/opencollective-frontend/blob/e00c9cc54c28b97a3aa33022cf02bfc8e377b71c/lib/theme/breakpoints.js#L1). So you can achieve responsiveness with something like this:

```jsx
<!-- Order will be 2 under 40em, then 1 above 40em -->
<StyledButton order={[2, 1]} />